gef-ui/templates/application-details-comments.njk

```diff
@@ -0,0 +1,72 @@
+{% extends "index.njk" %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/character-count/macro.njk" import govukCharacterCount %}
+
+{% block pageTitle %}
+  Submit to be checked at your bank
+{% endblock %}
+
+{% block content %}
+
+  {% if errors %}
+    {{ govukErrorSummary({
+      titleText: "There is a problem",
+      errorList: errors.errorSummary,
+      attributes: {
+        'data-cy': 'error-summary'
+      }
+    }) }}
+  {% endif %}
+
+  <div class="govuk-grid-row" data-cy="application-submit-page">
+    <div class="govuk-grid-column-two-thirds-from-desktop">
+      <p class="govuk-!-margin-bottom-6">
+        <a class="govuk-link" href="/gef/application-details/{{applicationId}}" data-cy="back-link">Back to application</a>
+      </p>
+      <h1 class="govuk-heading-xl govuk-!-margin-bottom-6" data-cy="main-heading">
+        Submit to be checked at your bank
+      </h1>
+
+      <p class="govuk-body govuk-!-margin-bottom-6">
+        Someone at your bank must check your automatic inclusion notice before they can submit it to UKEF.
+      </p>
+    </div>
+
+    <div class="govuk-grid-column-one-half-from-desktop">
+      <form
+        method="post"
+        data-cy="form">
+
+        {{ govukCharacterCount({
+          name: "comment",
+          id: "comment",
+          value: comment,
+          maxlength: maxCommentLength,
+          label: {
+            text: "Comments (optional)",
+            classes: "govuk-body"
+          },
+          hint: {
+            text: "This will only be seen at your bank."
+          },
+          attributes: {
+            'data-cy': 'application-comments'
+          }
+        }) }}
+
+        <p class="govuk-button-group govuk-!-margin-top-7" style="vertical-align: middle">
+          {{ govukButton({
+            text: "Submit",
+            attributes: {
+              'data-cy': 'submit-button'
+            },
+            classes: "govuk-!-margin-right-6"
+          }) }}
+          <a class="govuk-link" href="/gef/application-details/{{applicationId}}" data-cy="cancel-link">Cancel</a>
```

Comment:
Do any status changes need to hapeen here?

Reply:
Im not aware of any ... its simply bailing out the submission (which changes the state)

Reply:
OK that makes sense to me